tsconfig.json

```diff
@@ -1,11 +1,8 @@
 {
   "compilerOptions": {
     "target": "es5",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "types": ["node"],
```

Comment:
I'm not sure what this option does. Will it affect types for other modules?

Reply:
I think even if I remove this types option it will still be the same. If we keep it it won't affect anything https://www.typescriptlang.org/tsconfig#what-does-this-affect